### PR TITLE
Mention `PostgreSQL with Promscale` as Jaeger span storage

### PR DESCRIPTION
**Pull request by arajkumar**
Promscale is known to Jaeger complaint[1] since v0.16.0[2].

[1] https://github.com/timescale/promscale/actions/runs/3293909868/jobs/5430887629#step:6:292-354 
[2] https://github.com/timescale/promscale/releases/tag/0.16.0

Signed-off-by: Arunprasad Rajkumar <ar.arunprasad@gmail.com>

## Which problem is this PR solving?
- 

## Short description of the changes
-

### CODE REVIEW

File: content/docs/next-release/spm.md [hunk 1]
```diff
@@ -157,6 +157,10 @@ The following configuration is required to enable the SPM feature:
   - Optional: Set `--prometheus.server-url` (or `PROMETHEUS_SERVER_URL` environment variable)
     to the URL of the prometheus server. Default: http://localhost:9090.
 
+{{< info >}}
+[PostgreSQL with Promscale](https://github.com/timescale/promscale#promscale-for-jaeger-and-opentelemetry) shall be used as an single storage backend for both [Prometheus metrics](https://docs.timescale.com/promscale/latest/send-data/prometheus/) and [Jaeger traces](https://docs.timescale.com/promscale/latest/send-data/jaeger/).
+{{< /info >}}
+
```
Comment (ramonguiu) [inline content/docs/next-release/spm.md:163]: I think this is too aggressive. It feels like pushing Promscale too much in their docs. I don't think it makes sense and I would keep out of this PR. We could consider doing it in the future.

File: content/docs/next-release/spm.md [hunk 2]
```diff
@@ -281,4 +285,4 @@ both ingress and egress spans in the `server` and `client` span kinds, respectiv
 [prom-metric-labels]: https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels
 [http-api-readme]: https://github.com/jaegertracing/jaeger/tree/main/docker-compose/monitor#http-api
 [spanmetrics-config-dimensions]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/spanmetricsprocessor/testdata/config-full.yaml#L46
-[spanmetrics-config-latency]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/spanmetricsprocessor/testdata/config-full.yaml#L38
+[spanmetrics-config-latency]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/spanmetricsprocessor/testdata/config-full.yaml#L38
```
Comment (ramonguiu) [inline content/docs/next-release/spm.md:288]: What has changed here?

File: content/docs/next-release/features.md [hunk 3]
```diff
@@ -81,4 +83,4 @@ traces belonging to these services/operations.
 
 See [Service Performance Monitoring (SPM)](../spm) for more details.
 
-[otlp]: https://opentelemetry.io/docs/reference/specification/protocol/
+[otlp]: https://opentelemetry.io/docs/reference/specification/protocol/
```
Comment (ramonguiu) [inline content/docs/next-release/features.md:86]: What has changed here?

File: content/docs/next-release/_index.md [hunk 4]
```diff
@@ -74,4 +77,4 @@ See [running a docker all in one image](getting-started#all-in-one).
 - [Using Jaeger with Envoy](https://www.envoyproxy.io/docs/envoy/latest/start/sandboxes/jaeger_tracing.html) (docs)
 
 [dapper]: https://research.google.com/pubs/pub36356.html
-[ubeross]: http://uber.github.io
+[ubeross]: http://uber.github.io
```
Comment (ramonguiu) [inline content/docs/next-release/_index.md:80]: What has changed here?

File: content/docs/next-release/_index.md [hunk 5]
```diff
@@ -30,7 +30,8 @@ Uber published a blog post, [Evolving Distributed Tracing at Uber](https://eng.u
 
   * [OpenTracing](http://opentracing.io/)-inspired data model
   * Uses consistent upfront sampling with individual per service/endpoint probabilities
-  * Multiple storage backends: Cassandra, Elasticsearch, memory.
+  * Multiple storage inbuilt backends: Cassandra, Elasticsearch, in-memory
+  * Community supported external storage backend via gRPC plugin: [PostgreSQL with Promscale](https://github.com/timescale/promscale#promscale-for-jaeger-and-opentelemetry), [ClickHouse](https://github.com/jaegertracing/jaeger-clickhouse)
```
Comment (ramonguiu) [inline content/docs/next-release/_index.md:34]: I am not sure what is the best way to bring this here. If you think this is the best way, go for it.

File: content/docs/next-release/_index.md [hunk 6]
```diff
@@ -47,7 +48,9 @@ See [Features](./features/) page for more details.
     * [Elasticsearch 5.x, 6.x, 7.x](./deployment/#elasticsearch)
     * [Kafka](./deployment/#kafka)
     * memory storage
-
+    * grpc-plugins:
```
Comment (ramonguiu) [inline content/docs/next-release/_index.md:51]:
```suggestion
    * certified grpc-plugins:
```

File: content/docs/next-release/_index.md [hunk 7]
```diff
@@ -30,7 +30,8 @@ Uber published a blog post, [Evolving Distributed Tracing at Uber](https://eng.u
 
   * [OpenTracing](http://opentracing.io/)-inspired data model
   * Uses consistent upfront sampling with individual per service/endpoint probabilities
-  * Multiple storage backends: Cassandra, Elasticsearch, memory.
+  * Multiple storage inbuilt backends: Cassandra, Elasticsearch, in-memory
```
Comment (alejandrodnm) [inline content/docs/next-release/_index.md:33]:
```suggestion
  * Multiple storage backends: Cassandra, Elasticsearch, in-memory
```

File: content/docs/next-release/spm.md (hunk 2 above)
Comment (arajkumar) [reply content/docs/next-release/spm.md]: Looks like \n is automatically by editor.

File: content/docs/next-release/_index.md (hunk 7 above)
Comment (yurishkuro) [reply content/docs/next-release/_index.md:33]: built-in

File: content/docs/next-release/_index.md [hunk 8]
```diff
@@ -30,7 +30,8 @@ Uber published a blog post, [Evolving Distributed Tracing at Uber](https://eng.u
 
   * [OpenTracing](http://opentracing.io/)-inspired data model
   * Uses consistent upfront sampling with individual per service/endpoint probabilities
-  * Multiple storage backends: Cassandra, Elasticsearch, memory.
+  * Multiple storage built-in backends: Cassandra, Elasticsearch, in-memory
+  * Community supported external storage backend via gRPC plugin: [PostgreSQL with Promscale](https://github.com/timescale/promscale#promscale-for-jaeger-and-opentelemetry), [ClickHouse](https://github.com/jaegertracing/jaeger-clickhouse)
```
Comment (yurishkuro) [inline content/docs/next-release/_index.md:34]:
```suggestion
  * Multiple built-in and community-supported storage backends
```
Let's de-dupe this page, this is already mentioned ~L50

**Review by yurishkuro — APPROVED** [review]

**Pull request merged**